app/controllers/idv/image_uploads_controller.rb

```diff
@@ -20,16 +26,23 @@ def create
 
         if client_response.success?
           doc_pii_form_response = Idv::DocPiiForm.new(client_response.pii_from_doc).submit
-          form_response = form_response.merge(doc_pii_form_response)
+          analytics.track_event(
+            Analytics::IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION,
+            doc_pii_form_response.to_h.merge(user_id: user_uuid),
+          )
           store_pii(client_response) if client_response.success? && doc_pii_form_response.success?
+
+          # merge the doc_pii_form_response for the presenter below
+          image_form_response = image_form_response.merge(doc_pii_form_response)
```

Comment:
maybe now that we're logging all these separate events, what if we just passed all 3 separate form responses to the presenter? instead of having to merge 2?

Reply:
Actually, we are only passing one response to the presenter, and the current logic for that requires the form responses merged.  They could be merged in the presenter_response method; a bit uglier, but more local to where it is needed. Sadly, they do need to be merged if the image_form_response succeeds but the doc_pii_form_response fails since the `remaining_attempts` are in the image_form_response.

```
def presenter_response(image_form_response, client_response, doc_pii_form_response)
    form_response = image_form_response
    if doc_pii_form_response.present?
        form_response = image_form_response.merge(doc_pii_form_response)
    end

    return client_response if form_response.success? && client_response.present?
    form_response
end
```